tap_pipedrive/tap.py

```diff
@@ -7,6 +7,7 @@
 from singer import set_currently_syncing, metadata
 from singer.catalog import Catalog, CatalogEntry, Schema
 from .config import BASE_URL, CONFIG_DEFAULTS
+from .config import BASE_OAUTH_URL, CONFIG_DEFAULTS
```

Comment:
Can we merge this and the previous line?
```suggestion
from .config import BASE_OAUTH_URL, BASE_URL, CONFIG_DEFAULTS
```